docs/API.md

```diff
@@ -382,6 +386,39 @@ View primitives
 </Document>
 ```
 
+## Hooks
+
+### `useWindowDimensions()`
+
+Returns the window dimensions of the parent `<Artboard>`. Returns `{ width: number, height: number, fontScale: number, scale: number }`.
+
+#### Example
+
+```js
+import { Page, Artboard, View, Text, useWindowDimensions } from 'react-sketchapp';
+
+const HomePage = () => {
+  const { height, width } = useWindowDimensions();
+
+  return (
+    <View style={{ flex: 1 }}>
+      <View style={{ height, width }}>
+        <Text>Hello World</Text>
+      </View>
+      <View style={{ height, width, backgroundColor: 'blue' }} />
+    </View>
+  );
+};
+
+render(
+  <Page>
+    <Artboard viewport={{ width: 360, height: 640 }}>
```

Comment:
it could be interesting to add 2 Artboard with different viewports and show what `useWindowDimensions` will be in both cases (as a comment below it) - and perhaps showing an additional div in case the width is bigger than a certain breakpoint

Reply:
I've added a `devices` array and am mapping through that, and created a conditional `View` for tablet/desktop.

I've created a `useBreakpoint` hook that returns a `getStyle` function, like this: `getStyle(['column', 'row'])` – for column on mobile and row on tablet/desktop. Might be worth featuring in a new doc file, probably overkill for `API.md` example though.